anesthetic/plot.py

```diff
@@ -1038,6 +996,8 @@ def quantile_plot_interval(q):
                   '4sigma': 0.999936657516334,
                   '5sigma': 0.999999426696856}
         q = (1 - sigmas[q]) / 2
+    elif isinstance(q, int) and q >= 1:
+        q = (1 - erf(q / np.sqrt(2))) / 2
     if isinstance(q, float) or isinstance(q, int):
```

Comment:
This is a nice addition! Should we add q to the docstrings officially? Tests look good too.

Reply:
Added in [62f1d9c](https://github.com/williamjameshandley/anesthetic/pull/212/commits/62f1d9c928c108ac8b913865c0be1a7bea779293).